src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+// Define action types for adding and removing a book.
+export const ADD_BOOK = 'book-store/redux/ADD_BOOK';
+export const REMOVE_BOOK = 'book-store/redux/REMOVE_BOOK';
```

Comment:
- [ ] Kindly notice that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and const `REMOVE_BOOK = bookstore/books/REMOVE_BOOK`, you should put `books` instead of `redux`.